modules/b2b_logic/logic.c

```diff
@@ -4448,6 +4492,20 @@ int b2bl_bridge_msg(struct sip_msg* msg, str* key, int entity_no)
 		LM_ERR("Failed to get to or from from the message\n");
 		goto error;
 	}
+
+	memset(&ct_uri, 0, sizeof(struct sip_uri));
+	if (contact_user && parse_uri(to_uri.s, to_uri.len, &ct_uri) < 0) {
+		LM_ERR("Not a valid sip uri [%.*s]\n", to_uri.len, to_uri.s);
+		goto error;
+	}
+
+	local_contact = tuple->local_contact;
+	if (get_local_contact(msg->rcv.bind_address, &ct_uri.user, &local_contact) < 0)
+	{
+		LM_ERR("Failed to get received address\n");
+		local_contact = tuple->local_contact;
+	}
+
```

Comment:
The `local_contact` variable is never used after you build the Contact here, so this code is unnecessary.

Reply:
Thank you for reviewing. It's my mistake when I ported from few patches, I fixed this, it's used by real in code later.